### ISSUE TITLE
plan a journal cascade

### ISSUE THREAD

**Issue by lizzieinvancouver** [temporalecologylab/TreeRings#13]
If we write a paper it would be nice to know the plan of journals we'll submit to so we know the relative word lengths and other basic formatting. Based on the list of other papers that @afong3 put together I suggest we plan on:

1. MEE  https://besjournals.onlinelibrary.wiley.com/journal/2041210X
2. Dendrochronolgia https://www.sciencedirect.com/journal/dendrochronologia
3. Tree Ring Research https://bioone.org/journals/tree-ring-research/scope-and-details

Though some software or hardware specific journals could work too if better for someone career-wise or such.... @afong3 Take a look at these and see what the formatting is like and check costs (if they are open access, we may have UBC discounts that apply, check out: https://scholcomm.ubc.ca/open-access-publisher-discounts/ )

**Comment by afong3** [comment]
This is far overdue... but here is the cascade I've been thinking after doing some more targeted research. Please let me know what you think and if I should add any more info.

| Journal                                  | Link                                                                                                                       | Article Type                                                                                                      | Open Access Charge | Word Count | Author Guidelines                                                                                | Abstract Word Count | Interesting Additions                                                                                                |
| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- | ------------------ | ---------- | ------------------------------------------------------------------------------------------------ | ------------------- | -------------------------------------------------------------------------------------------------------------------- |
| Methods in Ecology and Evolution         | [Aims and scope](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/aims-and-scope/read-full-aims-and-scope) | Practical Tools                                                                                                   | $2600 USD          | 3000-4000  | [guidelines](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/author-guidelines) | 350                 |                                                                                                                      |
| HardwareX                                | [Aims and Scope](https://www.hardware-x.com/content/aims)                                                                  |                                                                                                                   | $660 USD           |            |                                                                                                  |                     |                                                                                                                      |
| Dendrochronologia                        | [Aims and scope](https://www.sciencedirect.com/journal/dendrochronologia/about/aims-and-scope)                             | Technical Notes                                                                                                   | $3210 USD          |            |                                                                                                  |                     |                                                                                                                      |
| Trees: Structure and Function            | [Aims and Scope](https://link.springer.com/journal/468/aims-and-scope)                                                     | Technological problems                                                                                            | $3490              |            |                                                                                                  |                     | email piovesan@unitus.it with a short summary prior to manuscript submission                                         |
| Tree Ring Research                       | [Submission instructions](https://www.treeringsociety.org/journal.html)                                                    |                                                                                                                   |                    |            |                                                                                                  |                     |                                                                                                                      |
| Designs                                  | [Aims and Scope](https://www.mdpi.com/journal/designs/about)                                                               | Open Source Design                                                                                                | $1600 CHF          |            | [guidelines](https://www.mdpi.com/journal/designs/instructions)                                  |                     |                                                                                                                      |
| Plant Methods                            | [Aims and scope](https://plantmethods.biomedcentral.com/submission-guidelines/aims-and-scope)                              | [Methodology](https://plantmethods.biomedcentral.com/submission-guidelines/preparing-your-manuscript/methodology) |                    |            |                                                                                                  |                     |                                                                                                                      |
| Frontiers in Plant Science               | [Mission and scope](https://www.frontiersin.org/journals/plant-science/about#about-scope)                                  |                                                                                                                   |                    |            |                                                                                                  |                     |                                                                                                                      |
| Journal of Open Hardware                 | [About](https://ojs.lib.uwo.ca/index.php/openhardware/about)                                                               | Hardware Metapaper                                                                                                | $0                 | 4500       |                                                                                                  | 250                 |                                                                                                                      |
| The Journal of Open Engineering          | [About](https://www.tjoe.org/about)        [Ethos](https://www.tjoe.org/pub/open-publishing-in-engineering/release/1)      |                                                                                                                   | $0                 |            |                                                                                                  |                     | Videos, gifs, animations allowed. Useful for user interaction. Submissions are published as pre preprint immediately |
| Frontiers in Ecology and the Environment | [Aims and Scope](https://esajournals.onlinelibrary.wiley.com/hub/journal/15409309/aims-and-scope/read-full-aims-and-scope) |                                                                                                                   |                    |            |                                                                                                  |                     |                                                                                                                      |
|                                          |                                                                                                                            |                                                                                                                   |                    |            |                                                                                                  |                     |                                                                                                                      |

Notes:

- [Open Source Hardware Association](https://www.oshwa.org/definition/)
-

**Comment by lizzieinvancouver** [comment]
@afong3 Nice! So the cascade goes from top to bottom? Can you check if UBC has any agreements to pay the open access fees for any of these journals (https://scholcomm.ubc.ca/open-access-publisher-discounts/ I think)?

**Comment by lizzieinvancouver** [comment]
@afong3 Can you confirm the plan is Methods in Ecology and Evolution to submit first and did we come up with a cascade for me to think of as I review your draft?

**Comment by afong3** [comment]
@lizzieinvancouver I can confirm the plan was to start with Methods in Ecology and Evolution first. The potential journals earlier in the thread were sorted from top to bottom by priority.

**Comment by lizzieinvancouver** [comment]
@afong3 What's the non-open access cost for the top 3 journals on the list?

**Comment by afong3** [comment]
@lizzieinvancouver 

1. MEE - As of 2022 all papers published to MEE are fully Open Access.
2. Hardware X - All publications are published Open Access.
3. Dendrochronologia - "Subscription - No publication fee charged to authors, and published articles are immediately available to subscribers." Does it being free sound correct? I couldn't find another dollar amount that wasn't OA.

**Comment by lizzieinvancouver** [comment]
@afong3 Right, I think you mentioned (1). We'll just have to hope to get support for those costs if we end up accepted there (the lab has a budget of $0 for OA)  ... when you go to submit you can tell me what they ask for about committing to costs and we can discuss then. 

I suggest we move Hardware X down the list.

Most journals don't charge to publish, it's only the OA takeover of publishing (which I do NOT like) that is causing these crazy costs so maybe we start at MEE, then go to Dendrochronologia...

See also:
https://scholcomm.ubc.ca/open-access-publisher-discounts/ and https://dynamicecology.wordpress.com/2024/04/29/the-state-of-academic-publishing-in-3-graphs-5-trends-and-4-thoughts/
